content/components/segmented-control.mdx

```diff
@@ -0,0 +1,426 @@
+      An optional caption may be used to provide additional context to help users make a selection or explain how what
+      effect the selection will have. Caption text should be as short as possible.
+    </Text>
+  </Box>
+  <img
+    src="https://user-images.githubusercontent.com/2313998/180623806-23b98c83-e870-4eed-b1f1-635e919a3d2d.png"
+    role="presentation"
+    width="456"
+  />
+</Box>
+
+See the [layout](#layout) section for ways to compose a segmented control with a label.
+
+## Interaction
+
+Selections are mutually exclusive. Once a selection has been made, it can only be deselected by making a new selection.
+
+A segmented control does not require a "Save" button to apply the selection. When a choice is selected, the selection immediately affects an object, state, or view.
+
+## Layout
+
+### Composing the control with a label and caption
+
+#### Single-column
+
+<img
+  src="https://user-images.githubusercontent.com/2313998/180624841-cc4e4bf9-098c-425c-ba84-7df8413c2055.png"
+  role="presentation"
+/>
+
+A single-column layout is the same way form controls (for example, text inputs) are layed out. The single-column layout is useful for preserving horizontal space.
+
+#### Two-column
+
+<Box display="flex" mb={3} flexWrap="wrap" sx={{gap: 3}}>
+  <img
+    src="https://user-images.githubusercontent.com/2313998/180624842-f3abb168-1d3d-4cdb-a167-2c78059653d5.png"
+    role="presentation"
+    width="456"
+  />
+  <img
+    src="https://user-images.githubusercontent.com/2313998/180624843-75a82099-b95f-41f6-b9a9-35adc3e1537e.png"
+    role="presentation"
+    width="456"
+  />
+</Box>
+
+A two-column layout can be used to:
+
+- Align the control with adjacent elements
+- Conserve vertical space
+- Group related controls in a way that is easy to scan
+
+### Fill available width
+
+<Box
+  as="figure"
+  display="flex"
+  alignItems="flex-start"
+  flexDirection={['column-reverse', 'column-reverse', 'column-reverse', 'column-reverse', 'row']}
+  m={0}
+  mb={3}
+  sx={{gap: 3}}
+>
+  <Text as="p" mt="0">
+    A segmented control's width can stretch each button evenly to fill the width of the control's parent. This option is
+    intended to make the segmented control easier to adapt into different layouts.
+  </Text>
+  <img
+    src="https://user-images.githubusercontent.com/2313998/180624849-a8d785f2-0a0c-42dd-915c-65e26a09eb64.png"
+    role="presentation"
+    width="456"
+  />
+</Box>
+
+### Adapting to narrow areas
+
+There are cases where a segmented control needs to be placed in a spot that is too narrow to accommodate all of its options. In these cases, the control can be adapted to fit in the narrow area without sacrificing usability.
+
+<Box
+  as="figure"
+  display="flex"
+  alignItems="center"
+  flexDirection={['column', 'column', 'column', 'column', 'row']}
+  m={0}
+  mb={3}
+  sx={{gap: 3}}
+>
+  <figcaption>
+    <Text as="h4" fontSize={2} fontWeight="bold" m={0}>
+      Convert to an action menu
```

Comment:
How can we make this behavior part of SegmentedControl? It might be the most common responsive adaptation, we should make it simple for whoever is using it :)

Reply:
This is already built in to the SegmentedControl React component by using the `variant` prop. This info is mostly for designers who are making mocks in Figma.